packages/ckeditor5-build-balloon-block/package.json

```
@@ -77,7 +79,7 @@
     "directory": "packages/ckeditor5-build-balloon-block"
   },
   "scripts": {
-    "build": "webpack --mode production",
+    "build": "webpack --mode production && tsc -p ./tsconfig.release.json",
```

```suggestion
    "build": "webpack --mode production",
    "postbuild": "tsc -p ./tsconfig.release.json"
```

Let's generate typings as post-hook. I would keep the rule "a single task for a single job" as much as possible.